src/lib/image-transform.ts

```diff
@@ -12,7 +12,7 @@ import {
   TransformData,
 } from './types';
 
-export class ImageTransform {
```

Comment:
@harshithad0703 No need to change for ImageTransform class. As we're directly using this class, better to use Pascal case here

Reply:
made changes